episodes/22-external.md

```diff
@@ -26,8 +20,40 @@ exercises: 25
+
+Your lesson documentation should contain the following information, which should be kept up to date.
+
+### Description of the project - `README` file
+
+`README.md` file (located in the root directory) is the first piece of documentation that people will land on when visiting your lesson repository. It should concisely explain what the lesson is about and who it is for, and contain links and pointers to further information, including:
+
+  - **rendered version of the lesson** - even though this is typically included in the `About` section of the repository, it is always useful to link to the URL where the rendered lesson is visible.
+  - **contact information** - include email addresses or mailing lists or other details on how to get in touch with the lesson maintainers.
+  - **contributing** - this is an opportunity to list what kinds of contribution are sought and how to get involved in lesson development for new contributors. You can provide more details in a separate `CONTRIBUTING.md` file within the repository’s root directory and link to it from `README`.
+  - **credits/acknowledgements** - make sure to credit those who have helped in the lesson's development or inspired it, and/or any resources/templates that you have reused. You can also link to a separate `AUTHORS.md` file within the repository’s root directory to list all the people who contributed to the lesson content (if this list starts to become too large to include in the `README` itself).
+  - **citation** - it’s a very good idea to specify a reference to an appropriate academic publication so others can cite the use of the lesson in their own publications and media. A common practice is to include the citation information in a separate `CITATION.md` file within the repository’s root directory and link to it from `README`.
+  - **licence** - a short description of and a link to the lessons’s licence typically contained in a separate `LICENSE.md` file within the repository’s root directory.
+
+### Instructions for contributors - `CONTRIBUTING.md`
 
+`CONTRIBUTING.md` is a file within the repository's root directory where you should provide detailed description of the ways people can send their contributions, what kinds of contribution will be credited and in what ways. For example, at what point would someone be listed as an author and how you will credit contributions that are not recorded in the commit history of the project. The latter is particularly important for newcomers who may not be proficient with the use of GitHub's features for collaborative work, such as issues, mentions, pull requests and code review, but could still provide valuable input and contributions.
```

Comment:
Lesson repositories already have a generic Carpentries `CONTRIBUTING.md` file. I think we should mention that, and describe how and why participants might want to modify it.

Reply:
Addressed vi d68d785.